src/foundry/client/pixi/webgl/shaders/samplers/base-sampler.d.mts

```diff
@@ -0,0 +1,215 @@
+     * }`
+     * ```
+     */
+    static CONTRAST: string;
+
+    /**
+     * Saturation adjustment
+     * @defaultValue
+     * ```js
+     * `// Computing saturated color
+     * if ( saturation != 0.0 ) {
+     *  vec3 grey = vec3(perceivedBrightness(changedColor));
+     *  changedColor = mix(grey, changedColor, 1.0 + saturation);
+     * }`
+     * ```
+     */
+    static SATURATION: string;
+
+    /**
+     * Exposure adjustment.
+     * @defaultValue
+     * ```js
+     * `if ( exposure != 0.0 ) {
+     *   changedColor *= (1.0 + exposure);
+     * }`
+     * ```
+     */
+    static EXPOSURE: string;
+
+    /**
+     * The adjustments made into fragment shaders.
+     * @defaultValue
+     * ```js
+     * `vec3 changedColor = baseColor.rgb;
+     * ${this.CONTRAST}
+     * ${this.SATURATION}
+     * ${this.EXPOSURE}
+     * baseColor.rgb = changedColor;`
+     * ```
+     */
+    static get ADJUSTMENTS(): string;
+
+    /**
+       * @defaultValue
+       * ```js
+       * `precision ${PIXI.settings.PRECISION_VERTEX} float;
+       * attribute vec2 aVertexPosition;
+       * attribute vec2 aTextureCoord;
+       * uniform mat3 projectionMatrix;
+       * varying vec2 vUvs;
+
+       * void main() {
+       *   vUvs = aTextureCoord;
+       *   gl_Position = vec4((projectionMatrix * vec3(aVertexPosition, 1.0)).xy, 0.0, 1.0);
+       * }`
+       * ```
+       */
+    static override vertexShader: string;
+
+    static override fragmentShader: string | ((...args: any[]) => string);
+
+    /**
+     * Batch default vertex
+     */
+    static batchVertexShader: string;
+
+    /**
+     * Batch default fragment
+     */
+    static batchFragmentShader: string;
+
+    /**
+     * @defaultValue
+     * ```javascript
+     * {
+     *  tintAlpha: [1, 1, 1, 1],
+     *  sampler: 0
+     * };
+     * ```
+     */
+    static override defaultUniforms: AbstractBaseShader.Uniforms;
+
+    /**
+     * Batch geometry associated with this sampler.
+     */
+    static batchGeometry: typeof PIXI.BatchGeometry;
+
+    /**
+     * The size of a vertice with all its packed attributes.
+     * @defaultValue `6`
+     */
+    static batchVertexSize: number;
+
+    /**
+     * Pack interleaved geometry custom function.
+     */
+    protected static _packInterleavedGeometry:
+      | ((
+          element: PIXI.IBatchableElement,
+          attributeBuffer: PIXI.ViewableBuffer,
+          indexBuffer: Uint16Array,
+          aIndex: number,
+          iIndex: number,
+        ) => void)
+      | undefined;
+
+    /**
+     * A prerender function happening just before the batch renderer is flushed.
+     */
+    protected static _preRenderBatch(): ((batchRenderer: typeof BatchRenderer) => void) | undefined;
+
+    /**
+     * A function that returns default uniforms associated with the batched version of this sampler.
+     * @remarks Foundry annotated this as abstract
+     */
+    //TODO: This is Odd. Foundry sets default value to {}. Revisit.
```

Comment:
Seems to be a core Foundry mistake. Put it in #2618. It's not exactly a documentation issue, it's kind of just a mistake by them, but it isn't important enough to raise right now I don't think.